ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -77,13 +77,13 @@ object Cli {
         .action { (_, c) =>
           setTimeProviderType(c, TimeProviderType.Static)
         }
-        .text("Use static time, configured with TimeService through gRPC.")
+        .text("Use static time. When not specified, wall-clock-time is used.")
```

Comment:
I’ve removed the `configured with TimeService through gRPC` part since I don’t think that’s helpful in any way. Happy to add it back if you disagree.